packages/workflow/src/Interfaces.ts

```diff
@@ -1643,6 +1643,13 @@ export interface INodeType {
 			[method in WebhookSetupMethodNames]: (this: IHookFunctions) => Promise<boolean>;
 		};
 	};
+	nonRoutingOperations?: {
+		[resource: string]: {
+			[operation: string]: (
+				context: IExecuteFunctions,
+			) => Promise<INodeExecutionData[][] | NodeExecutionWithMetadata[][] | null>;
```

Comment:
This type is repeated and a bit obscure. Would it be clearer to have and use `type NodeOutput = INodeExecutionData[][] | NodeExecutionWithMetadata[][] | null`?